app/index.js

```diff
@@ -112,9 +112,6 @@ loadAppStatePromise.then((initialImmutableState) => {
   CrashHerald.init(getSetting(SEND_CRASH_REPORTS, initialSettings))
 
   telemetry.setCheckpointAndReport('state-loaded')
-  if (getSetting(HARDWARE_ACCELERATION_ENABLED, initialSettings) === false) {
```

Comment:
I think we still need to handle the case here where the local state pref and appstate pref are out of sync. I'll add an api to check the current value in muon